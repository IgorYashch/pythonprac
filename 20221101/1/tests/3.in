a, b, c = Omnibus(), Omnibus(), Omnibus()


a.b = b.b = c.b = True
a.a = b.a = c.a = "hello"

print(a.a, a.b)

print(a.b == b.b)

del c.b

print(a.b != a.c)