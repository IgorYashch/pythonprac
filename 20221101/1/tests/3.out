3 3
True
True
